docs/cep/table/index.md

```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
```

Comment:
@diana-macrometa - This is a syntax sugar for `replication.type`.

Reply:
Updated.